_sass/custom/_blog.scss

```diff
@@ -192,4 +192,8 @@
   text-align: center;
   width: fit-content;
   margin: map-get($spacers, 2) auto 0;
+}
+
+.italic{
```

Comment:
There is a `.font-italic` style class already, please use this instead of adding a new style